### PR TITLE
Code updated for Venmo transfer method - DTSERWTHRE-33

### PR DESCRIPTION
**Pull request by nanbehera**
Code updated for Manage Venmo transfer method

### CODE REVIEW

**Review by aparthibanpaypal — DISMISSED** [review]
Reviewed.

**Review by pdamodarannair — APPROVED** [review]
Approved

File: Common/Sources/Extensions/HyperwalletTransferMethod.swift [hunk 1]
```diff
@@ -42,6 +42,12 @@ extension HyperwalletTransferMethod: GenericCellConfiguration {
                           "to".localized(),
                           getField(TransferMethodField.email.rawValue) ?? "")
 
+        case "VENMO_ACCOUNT":
```
Comment (jsaini-hw) [inline Common/Sources/Extensions/HyperwalletTransferMethod.swift:45]: Can you update this to `TransferMethodType.venmoAccount.rawValue`. Also update other hardcode values for all cases with their respective enum rawtype as well.

File: TransferMethodRepository/Sources/TransferMethodRepository.swift [hunk 2]
```diff
@@ -73,6 +73,10 @@ final class RemoteTransferMethodRepository: TransferMethodRepository {
             Hyperwallet.shared.createPayPalAccount(
                 account: payPalAccount,
                 completion: TransferMethodRepositoryCompletionHelper.performHandler(completion))
+        } else if let venmoAccount = transferMethod as? HyperwalletVenmoAccount {
+            Hyperwallet.shared.createVenmoAccount(
+                account: venmoAccount,
+                completion: TransferMethodRepositoryCompletionHelper.performHandler(completion))
```
Comment (jsaini-hw) [inline TransferMethodRepository/Sources/TransferMethodRepository.swift:79]: Update TransferMethodRepositoryTests class to add unit tests for venmo account

Comment (jsaini-hw) [reply TransferMethodRepository/Sources/TransferMethodRepository.swift:79]: Also update `ListTransferMethodSuccessResponse.json` response file to return Venmo account and then update `testListTransferMethods_returnsBankAccount` to assert venmo account

File: Common/Sources/Extensions/HyperwalletTransferMethod.swift (hunk 1 above)
Comment (nanbehera) [reply Common/Sources/Extensions/HyperwalletTransferMethod.swift:45]: Done

Comment (nanbehera) [reply Common/Sources/Extensions/HyperwalletTransferMethod.swift]: Code updated and pushed

File: TransferMethodRepository/Sources/TransferMethodRepository.swift (hunk 2 above)
Comment (nanbehera) [reply TransferMethodRepository/Sources/TransferMethodRepository.swift:79]: code updated

**Comment by ddhar-hw** [comment]
Kindly fix the failing tests on your branch

**Review by ddhar-hw — APPROVED** [review]

**Review by aparthibanpaypal — APPROVED** [review]
Approved

**Review by pdamodarannair — APPROVED** [review]
Approved

**Review by aparthibanpaypal — DISMISSED** [review]

**Review by jsaini-hw — COMMENTED** [review]
@nmadshetty  This ticket is to add Venmo Transfer Method and deactivate Transfer Method and I do not see UI tests for both

File: UITests/Responses/TransferMethod/ListTransferMethodResponse.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
         },
         {
             "token": "trm-11111111-0000-0000-0000-000000000000",
-            "type": "BANK_ACCOUNT",
+            "type": "VENMO_ACCOUNT",
```
Comment (jsaini-hw) [inline UITests/Responses/TransferMethod/ListTransferMethodResponse.json:45]: This json doesn't match Venmo account response, Venmo account does not have bank id, branch id, bankAccountId etc fields

File: UITests/TransferMethod/AddTransferMethodVemoAccountTests.swift [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import XCTest
+
+class AddTransferMethodVemoAccountTests: BaseTests {
+    var selectTransferMethodType: SelectTransferMethodType!
+    var addTransferMethod: AddTransferMethod!
+    let bankAccount = NSPredicate(format: "label CONTAINS[c] 'Bank Account'")
```
Comment (jsaini-hw) [inline UITests/TransferMethod/AddTransferMethodVemoAccountTests.swift:6]: It should not be Bank Account but Venmo

File: UITests/TransferMethod/AddTransferMethodVemoAccountTests.swift [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    override func setUp() {
+        super.setUp()
+
+        app = XCUIApplication()
+        app.launchEnvironment = [
+            "COUNTRY": "US",
+            "CURRENCY": "USD",
+            "ACCOUNT_TYPE": "VENMO_ACCOUNT",
+            "PROFILE_TYPE": "BUSINESS"
```
Comment (jsaini-hw) [inline UITests/TransferMethod/AddTransferMethodVemoAccountTests.swift:16]: Use Profile type INDIVIDUAL

**Review by jsaini-hw — APPROVED** [review]

**Review by pdamodarannair — APPROVED** [review]

**Review by ddhar-hw — APPROVED** [review]

**Pull request merged**